en/case-studies/rwanda-rural-water.rst

```diff
@@ -0,0 +1,111 @@
+Data collection by QGIS/QField for O&M work of rural water supply systems in Rwanda
+=========================================================================================================
+
+By Rural Water and Sanitation Services Department(RWSS), Water and Sanitation Corporation (`WASAC <https://wasac.rw>`__), Rwanda
+
+Goal
+----
+
+To conduct data collection of all rural water supply network in Rwanda, 
+and keep updating the data continuously in order to improve operation & 
+maintenance of waterworks.
+
+Project preparation
+-------------------
+
+Before starting our data collection, we conducted the following things.
+ - Develop our own PostGIS database
+ - Develop QGIS project template with Geopackage. The Geopackage table design should be equal to PostGIS. So we can just copy and paste to PostGIS.
+
+Apart from preparing Android devices, if possible, you may purchase GPS devices. In WASAC, we bought Garmin GPSMAP 64S. 
+Sometimes, GPS of smartphone and tablet is not very accurate, so we normally capture the same location by using Garmin GPS, then correct the location of QField's data after data collection work.
+
+Data collection
+---------------
+
+Once we prepared Geopackage and QGIS project template, 
+we conducted training of QGIS/QField in July 2018 and 
+launched our data collection work in 27 districts in the whole country of Rwanda. 
+27 engineers sent their Geopackage to central office in Kigali, 
+the MIS specialist validated and entered their data from Geopackage to PostGIS database. 
```

Comment:
What is the "MIS specialist"? Has this been introduced before?

Reply:
It is a position in WASAC. MIS (Management Information System) Specialist. Let me add it.

Reply:
I added description of MIS in this commit(https://github.com/opengisch/QField-docs/pull/77/commits/5d15d0943a202e52235bb906d356519148aa2030).